pkgs/development/tools/jdt-ls/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, stdenv
+, fetchurl
+, jdk
+, bash
+}:
+
+stdenv.mkDerivation rec {
+  pname = "jdt-ls";
+  version = "0.67.0";
+  timestamp = "202012170459";
+
+  src = fetchurl {
+    url = "https://download.eclipse.org/jdtls/milestones/${version}/jdt-language-server-${version}-${timestamp}.tar.gz";
+    sha256 = "11qzwf2l737hdv1g7bfcrmdxcrl7g9z2k9nhy9lknlabmhq4mcdl";
+  };
+
+  # After running unpackPhase, the generic builder changes the current directory
+  # to the directory created by unpacking the sources. If there are multiple
+  # source directories, you should set sourceRoot to the name of the intended
+  # directory.
+  sourceRoot = ".";
+
+  buildInputs = [
+    bash
+    jdk
+  ];
+
+  installPhase =
+    let
+      # There are config_ss_* directories too, use them to run the lightweight
+      # version, see
+      # https://github.com/eclipse/eclipse.jdt.ls/wiki/Running-the-lightweight-syntax-language-server
+      config-path = if stdenv.isDarwin then "config_mac" else "config_linux";
+    in
+    ''
+      JARs_dir=$out/share/plugins
```

Comment:
It's strange that you have `$out/share/plugins` and I have `$out/share/java/plugins`. Do they both work? What is the difference between these two directories?

Reply:
I think it doesn't matter, they just need to be in the same directory. Did you pick `share/java/plugins` for a specific reason?
In the `config.ini` file, we have something like this: `osgi.bundles=reference\:file\:....jar` As I understood it, that jars are relative to the launcher jar, the one in the `java ... -jar file.jar` command

Reply:
``share/java/plugins`` would be a better idea because it is java specific and it is clear in a combined environment where the path is coming from.